src/cli.ts

```diff
@@ -88,6 +91,16 @@ const freezeOpt: opt = [
   { default: false },
 ];
 const silentOpt: opt = ["--silent", "Silence all output", { default: false }];
+const buildConfigOpt: opt = [
+  "--build-config <file>",
+  "Path to a rollup config file",
+  {},
+];
+const buildOutputOpt: opt = [
+  "--build-output <file>",
```

Comment:
Rollup outputs a directory not a single file. I think it's important to make the directory case the default. Also `--output / -o <dir>` . Let's remove the file case entirely.